server/chat-plugins/mafia.ts

```diff
@@ -704,8 +708,12 @@ class Mafia extends Rooms.RoomGame<MafiaPlayer> {
 		if (!(target in this.playerTable) && target !== 'novote') {
 			return this.sendUser(userid, `|error|${target} is not a valid player.`);
 		}
+		if (!this.votingAll) return this.sendUser(userid, `|error|Voting is not allowed.`);
```

Comment:
Would be wise to move this to the very top of the method so you error out sooner. Theres no need to perform the other checks first in this case.